docs/governance/tokenomics.md

```diff
@@ -2,200 +2,178 @@
 
 ```shell
 cudos-noded tx slashing unjail --from mykey [flags]
 ```
 
+### Unbonding period
 
-For each validator, jailing starts after signed_blocks_window + start_height many blocks.
-We have signed_blocks_window = 19200.
+If a validator is jailed or leaves the bonded validators set, it enters an **unbonding period**. A validator that has failed to get itself back into the bonded validators set *within 21 days* is said to be **unbonded**.
 
-This means that the earliest a validator can be jailed is 19200 blocks after they join the network.
-
-Afterwards, they will get jailed as soon as they miss 17280 blocks in any interval of 19200 blocks.
-
-If a validator signs more than one block at the same height, that's also an offence punishable by slashing 5% of their tokens.
-
-This could be the case if someone duplicates their validator and try to double their voting power.
+:::caution 
+Slashing can still occur during the unbonding period if there is evidence of bad behaviour during the period when a Validator was bonded.
+:::
 
-Slashed tokens are sent to the community pool.
+### Token holder delegating
 
+Anyone holding CUDOS tokens can delegate tokens to one or more **validator nodes** of their choice. Validators and Delegators then share rewards **based on the amounts they have contributed to the pool**.
 
-## Transactions 
+Validator Operators are free to set their own **commission rate** to charge for delegating. The **commission rate** can change at most once daily. However, a validator's **commission rate** CANNOT exceed the **maximum commission rate** set when the validator node was first initiated. Commission fees can be viewed under [Validator details in the Cudos Explorer](https://explorer.cudos.org/validators). 
 
-How to view the mempool? Or pending transactions 
-If you're a validator you can keep your mempool to yourself when it is your turn to propose a block then you can submit your own transactions. 
+![validator-details](@site/static/img/validator-details.png)
 
-If you sync your own node with other nodes then you can view pending transactions.
+### Redelegation
 
-* Tx fees are calculated based on the **gas_wanted value rather than gas_used**. Therefore users need to be aware of this and should not provide extra gas_wanted values “just in case”.
+1. A **Delegator** can **redelegate** tokens from **Validator A** to **Validator B** without incurring any penalties or extra waiting times. A redelegation maturation period now follows for 21 days. 
 
-* We have a min-gas-price parameter set at 0.000005 CUDOS below which txs won’t be accepted into mempools
+2. Following on from the above example, if a **Delegator** wishes to **redelegate** their tokens from **Validator B** to a hypothetical **Validator C** they must wait *21 days*.
 
-    This parameter is local to each node on the network, can be changed off-chain, and cannot be queried on-chain.
+3. A **Validator Operator** can redelegate funds to an *existing* node in two ways:
```

Comment:
It doesn't make sense for a validator to redelegate tokens because then they would be paying commission to that other validator as well as losing voting power. Moreover, the two ways mentioned here are commonly available to everyone. Hence I'd probably remove this part as I don't think it's very relevant to a tokenomics doc.

Reply:
done